rules/modules.go

```diff
@@ -0,0 +1,92 @@
+package rules
```

Comment:
I'd like to rename `modules.go` to `module_source.go`.

Reply:
Updated